<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWindow</class>
 <widget class="QWidget" name="MusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/BackButtonIcon.png</normaloff>:/BackButtonIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>105</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="SongList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>541</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="VolumeUp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VolumeDown">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources/biobloom.qrc"/>
 </resources>
 <connections/>
</ui>
